src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -625,13 +625,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Function IsMethodLevelMember(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsMethodLevelMember
-            If TypeOf node Is MethodBlockBaseSyntax AndAlso
-                Not TypeOf node.Parent Is PropertyBlockSyntax AndAlso
-                Not TypeOf node.Parent Is EventBlockSyntax Then
+
+            If TypeOf node Is MethodStatementSyntax AndAlso Not TypeOf node.Parent Is MethodBlockBaseSyntax Then
```

Comment:
Will this work? The tree traversing thing above where you go up the nodes until you see the declaration uses this helper, but if you start in a LocalDeclaration in a regular VB ```Sub``` method, you'd see the LocalDeclaration, and then the SubBlock (not the SubStatement)...